card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -54,7 +54,14 @@ def form_for_multi
 
   def subcard_input_names
     return "" if !form_root_format || form_root_format == self
-    "#{@parent.subcard_input_names}[subcards][#{card.contextual_name}]"
+    "#{@parent.subcard_input_names}[subcards][#{name_in_form}]"
+  end
+
+  # If you use subfield cards to render a form for a new card
+  # then the subfield cards should be created on the new card not the existing
+  # card that build the form
+  def name_in_form
+    @native_mode == :new ? card.relative_name : card.contextual_name
```

Comment:
More precisely and less sarcastic: I remember that we have touched that before during a frustrating debug session. So my guess is if we change that and we are lucky that our tests will discover the problems caused by that change then it's a few hours of debugging. If we are unlucky we will notice weird stuff in a few months and spend a few days to find and fix it.  Maybe I'm exaggerating. I agree that in principle in an ideal code base the relative name should work but it is sitting at the gate to the hell of unpredictable side effects, the subcards handling :scream: